### ISSUE TITLE
Describing associations in SCAN where the associate taxa is only "observed"

### ISSUE THREAD

**Issue by neilcobb** [ParasiteTracker/data-issues-observations-and-questions#2]
To date it appears that 
Fields used to list associated Taxa with the specimen that is cataloged
associatedTaxa
habitat
substrate

Field used to describe the association (although the association may be described in the field used to identify the association
dynamicProperties
--

**Comment by seltmann** [comment]
@neilcobb I agree, and also in the dwc:dynamicProperties field, which may bring in data in json format if it is done by the dwc standard. I have also entered data in this field in simple key:value pairs.

**Comment by seltmann** [comment]
May also come in completely unstructured in the notes field.

**Comment by neilcobb** [comment]
Regardless, PT allows us to really focus on data few people paid attention to
